hq/app/controllers/HQController.scala

```diff
@@ -40,11 +40,12 @@ class HQController(val config: Configuration)
   }
 
   def generateCredentialsReport(accountId: String) = AuthAction.async {
+    val delay = 3.seconds
     attempt {
       for {
         account <- AWS.lookupAccount(accountId, accounts)
         client = IAMClient.client(account)
-         _ <- Retry.until(IAMClient.generateCredentialsReport(client), CredentialsReport.isComplete, "Failed to generate credentials report")
+         _ <- Retry.until(IAMClient.generateCredentialsReport(client), CredentialsReport.isComplete, "Failed to generate credentials report", delay)
```

Comment:
This is *borderline* too complicated for the controller and I think the `delay` constant pushes it over the edge for my taste. This line's logic and string constants are specific to the credentials report and the IAM client's functionality. I'd probably prefer to put them in a method on the IAMClient object so that the controller can just ask for a result without needing to know about how it worked.

Reply:
will move to IAMClient